HealthSDK/GiniHealthSDKExample/GiniHealthSDKExample/InvoicesList/InvoicesListViewModel.swift

```diff
@@ -9,37 +9,77 @@ import UIKit
 import GiniHealthAPILibrary
 import GiniCaptureSDK
 import GiniBankAPILibrary
+import GiniHealthSDK
+
+struct Bank: Codable {
+    let name: String
+    let iconName: String
+    let accentColor: GiniHealthSDK.Color
+    let textColor: GiniHealthSDK.Color
+
+    init() {
+        name = "Sparkasse"
+        iconName = "sparkasseBankIcon"
+        accentColor = GiniHealthSDK.Color(uiColor: .red)
+        textColor = GiniHealthSDK.Color(uiColor: .white)
+    }
+
+    internal init(name: String, 
+                  iconName: String,
+                  accentColor: Color,
+                  textColor: Color) {
+        self.name = name
+        self.iconName = iconName
+        self.accentColor = accentColor
+        self.textColor = textColor
+    }
+}
 
 struct DocumentWithExtractions: Codable {
     var documentID: String
     var amountToPay: String?
     var paymentDueDate: String?
     var recipient: String?
-    
-    init(documentID: String, extractionResult: GiniHealthAPILibrary.ExtractionResult) {
+    var isPayable: Bool?
+    // TODO: - Will be replace in next task with real data
+    var bank: Bank
+
+    init(documentID: String, extractionResult: GiniHealthAPILibrary.ExtractionResult, bank: Bank) {
         self.documentID = documentID
         self.amountToPay = extractionResult.payment?.first?.first(where: {$0.name == "amount_to_pay"})?.value
         self.paymentDueDate = extractionResult.extractions.first(where: {$0.name == "payment_due_date"})?.value
         self.recipient = extractionResult.payment?.first?.first(where: {$0.name == "payment_recipient"})?.value
+        self.isPayable = !(extractionResult.payment?.first?.first(where: {$0.name == "iban"})?.value.isEmpty ?? true)
+        self.bank = bank
     }
     
-    init(documentID: String, extractions: [GiniBankAPILibrary.Extraction]) {
+    init(documentID: String, extractions: [GiniBankAPILibrary.Extraction], bank: Bank) {
         self.documentID = documentID
         self.amountToPay = extractions.first(where: {$0.name == "amount_to_pay"})?.value
         self.paymentDueDate = extractions.first(where: {$0.name == "payment_due_date"})?.value
         self.recipient = extractions.first(where: {$0.name == "payment_recipient"})?.value
+        self.isPayable = !(extractions.first(where: {$0.name == "iban"})?.value.isEmpty ?? true)
```

Comment:
[Request] The "is payable" check should be done via `GiniHealth.checkIfDocumentIsPayable(docId:)`. This is an asynchronous call so it should happen outside of the initialiser. 

[Suggestion]
I can think of calling `checkIfDocumentIsPayable` after receiving the extractions and then storing `isPayable` in the data model of the invoice next to the extractions.

Alternatively it could be called whenever the invoice cell is displayed for the first time. This would show the most up-to-date `isPayable` state, but could trigger many requests and also a loading indicator should be displayed while the payable state is loaded.

@zladzeyka What do you think?